software/containers_on_frontier.rst

```diff
@@ -122,12 +122,68 @@ This example will look at building a container image for testing collective comm
+    | 23.12 | gnu - [ ``GCC@12.3.0``,  ``MPICH@3.4.3``,  ``ROCM@5.7.1`` ]    | Ubuntu     | savannah.ornl.gov/olcf-container-images/frontier/ubuntu/gnu/cpe:23.12      |
+    |       |                                                                +------------+----------------------------------------------------------------------------+
+    |       |                                                                | OpenSUSE   | savannah.ornl.gov/olcf-container-images/frontier/opensuse/gnu/cpe:23.12    |
+    |       |                                                                +------------+----------------------------------------------------------------------------+
+    |       |                                                                | RockyLinux | savannah.ornl.gov/olcf-container-images/frontier/rockylinux/gnu/cpe:23.12  |
+    +-------+----------------------------------------------------------------+------------+----------------------------------------------------------------------------+
+
+..
+    For later use.
+    |       | cray - [ ``LLVM@17.0.6``,  ``MPICH@3.4.3``,  ``ROCM@5.7.1`` ]  | Ubuntu     | savannah.ornl.gov/olcf-container-images/frontier/ubuntu/cray/cpe:23.12     |
+    |       |                                                                +------------+----------------------------------------------------------------------------+
+    |       |                                                                | OpenSUSE   | savannah.ornl.gov/olcf-container-images/frontier/opensuse/cray/cpe:23.12   |
+    |       |                                                                +------------+----------------------------------------------------------------------------+
+    |       |                                                                | RockyLinux | savannah.ornl.gov/olcf-container-images/frontier/rockylinux/cray/cpe:23.12 |
+    +-------+----------------------------------------------------------------+------------+----------------------------------------------------------------------------+
+
+Apptainer Modules
+^^^^^^^^^^^^^^^^^
+.. warning::
+
+    The modules described in this section are experimental!
+
+To make the use of apptainer easier, OLCF provides some modules that automatically bind in the needed libraries to run
+apptainer with the host mpi and rocm. To accsess these modules load ``olcf-container-tools``. You should then see two
+modules ``apptainer-enable-mpi`` and ``apptainer-enable-gpu``.
+
+Example Workflow
+^^^^^^^^^^^^^^^^
+To see how one might use these containers and modules we have an example of building and running lammps. You can
+find examples for cpu and gpu lammps runs `here <https://github.com/olcf/olcf_containers_examples/tree/main/frontier/containers_on_frontier_docs/apptainer_wrappers_lammps>`_.
+Clone the git repo on any ``x86_64`` machine, navigate to the correct folder and run:
```

Comment:
User can build the container with no problem on Frontier as well, so indicate that adjust the wording that says "after the image is built transfer it to Frontier". So you could write something like:

```
Clone the git repo onto Frontier (or any ``x86_64`` machine), navigate to the correct folder and run:
.. code-block::
    apptainer build lammps.sif lammps.def
After the image is built transfer it to Frontier if its on another machine, and run it by submitting the ``submit.slurm`` batch script that accompanies it.

Reply:
Fixed in [b1069b3](https://github.com/olcf/olcf-user-docs/pull/870/commits/b1069b3a71210e8864cc6eeb20f70d7e5c8e53e7)